agilesix/diffusion-marketplace
# DM-2430: Renamed Vamc table to VaFacility and changed all associated …

### JIRA issue link
https://agile6.atlassian.net/browse/DM-2430

## Description - what does this code do?
1. Renames `vamc` table in the DB to `va_facility`.
2. Changes `/vamc` endpoints to `/facilities`.
3. Changes all `vamc` files and calls in application to `va_facility` files and calls.
4. Changes `/visn` routes to `/visns`

## Testing done - how did you test it/steps on how can another person can test it 
_**The Setup**_
1. In your terminal, run `rails db:migrate`.
2. Open a new terminal window and run `rails c` and then run `Vamc.destroy_all`.
3. Go back to the original terminal and run `rails va_facilities:create_va_facilities_and_transfer_data`.
6. In the console terminal, run `VaFacility.all.count` and make sure you have 1211 records (the amount of entries in our vamc.json file).

_**Testing the Routes**_
1. Visit `/facilities` and make sure the page loads.
2. Visit `/facilities/any number from 1-1211` and make sure the page loads.
3. Visit `/facilities/pittsburgh` and make sure the page loads.
4. Visit `/facilities/1212` and make sure you get a routing error.
5. Visit `/visns` and make sure the page loads.
6. Visit `/visns/1` and make sure the page loads.
7. Visit `/visns/24` and make sure you get an error.

## Screenshots, Gifs, Videos from application (if applicable)
![Screen Shot 2021-03-18 at 3 12 08 PM](https://user-images.githubusercontent.com/34111449/111687123-4d1c2600-8800-11eb-9ef6-cf584ef11c88.png)
![Screen Shot 2021-03-18 at 3 47 52 PM](https://user-images.githubusercontent.com/34111449/111687965-4fcb4b00-8801-11eb-817e-1e26427e9f16.png)
![Screen Shot 2021-03-18 at 3 49 30 PM](https://user-images.githubusercontent.com/34111449/111688324-8acd7e80-8801-11eb-87db-a90b6c2e23e5.png)
![Screen Shot 2021-03-18 at 3 49 58 PM](https://user-images.githubusercontent.com/34111449/111688564-99b43100-8801-11eb-8da8-1dd992c12be6.png)
![Screen Shot 2021-03-18 at 3 50 17 PM](https://user-images.githubusercontent.com/34111449/111688757-a5075c80-8801-11eb-8fd0-531e42415d5c.png)

## Link to mock-ups/mock ups (image file if you have it) (if applicable)


## Acceptance criteria
- Everywhere it says VAMC it should say Facility
- This includes URL, breadcrumbs, page header, search header
- Change DB table

## Definition of done
